extensions/blocks/podcast-player/icons.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+function createSVGs() {
+	// Prevent repeated initialization.
+	if ( document.getElementById( 'jetpack-player-podcast-icons' ) ) {
+		return;
+	}
+
+	const svgTemplate = document.createElementNS( 'http://www.w3.org/2000/svg', 'svg' );
+	svgTemplate.id = 'jetpack-player-podcast-icons';
+	svgTemplate.classList.add( 'podcast-player-icons' );
```

Comment:
Comment about why we need this class would be good here.
cc @jeryj

Reply:
This class is necessary to fix a CSS bug in 2020 that assigns a `path` attribute that gets applied to our `svg` symbol definitions, that then gets passed into our `<use>` via the shadow dom, and we're not able to target it at that point. So, we needed a way to target just our `<svg>` symbol definitions to override this 2020 `path` rule.

Reply:
That said, we could target it via the `id` instead, if we want to be that heavy-handed about it.

Reply:
> we could target it via the id instead

I think we should do it. There's no need for an extra class then, no? Can we remove it?